addins/Cake.NDepend.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.NDepend
 NuGet: Cake.NDepend
-Prerelease: false
 Assemblies:
 - "/**/Cake.NDepend.dll"
-Repository: https://github.com/cake-contrib/cake-ndepend
+Repository: https://github.com/joaoasrosa/cake-ndepend
```

Comment:
Nuspec issue too
```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Cake.NDepend</id>
    <version>1.2.0</version>
    <authors>João Rosa</authors>
    <owners>João Rosa</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <licenseUrl>https://github.com/joaoasrosa/cake-ndepend/blob/master/LICENSE</licenseUrl>
    <projectUrl>https://github.com/joaoasrosa/cake-ndepend</projectUrl>
    <iconUrl>https://raw.githubusercontent.com/cake-contrib/graphics/a5cf0f881c390650144b2243ae551d5b9f836196/png/cake-contrib-medium.png</iconUrl>
    <description>Cake Build addin for NDepend.</description>
    <releaseNotes>See https://github.com/joaoasrosa/cake-ndepend/blob/master/ReleaseNotes.md</releaseNotes>
    <copyright>Copyright (c) 2018 João Rosa. MIT license</copyright>
    <tags>cake ndepend cake-build cake-contrib</tags>
    <repository type="git" url="https://github.com/joaoasrosa/cake-ndepend" />
    <dependencies>
      <group targetFramework=".NETFramework4.6.1" />
      <group targetFramework=".NETStandard2.0" />
    </dependencies>
  </metadata>
</package>
```

Maybe cake-contrib should be preferred if exists and not changed, as many projects won't push a packages immediately after transfer.

Reply:
@devlead There is logic in AddinDiscoverer to check if the project has been transferred to cake-contrib and, if so, favor the `https://github.com/cake-contrib/<projectname>` over `https://github.com/<authorname>/<projectname>`. This logic is dependent on the name of the NuGet package matching the name of the project under the cake-contrib organization.

In this case, the package is called `Cake.NDepend` but the project is called `cake-ndepend` (notice in one case there is a period and a dash in the other case). 

Possible solutions:
1. The author could fix the nuspec to include the URL pointing to the project under the cakecontrib-organization.
2. We could rename the project in the cakecontrib organization to match the package name

Reply:
Submitted a [PR](https://github.com/cake-contrib/cake-ndepend/pull/22) to resolve URLs.